ReactNative/Views/LogoView.swift

```diff
@@ -27,29 +27,13 @@ class LogoView: UIView {
         return reactView
     }()
 
-    override init(frame: CGRect) {
+    init(frame: CGRect, url: String) {
```

Comment:
Currently I don't know how to resize the React View correctly as component inside requires exact width and height. Lets discuss what we should do about that. (maybe I can teach it to autoscale, but then what about text/logo/border radius size?)